DigitalExcellence/dex-backend
# Feature/159 persistent identityserver

## Description

Changed the identity server to have a persistent data store.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

login on the frontend.
restart the identity server.
now login again
the api database should only have one user.

## Link to issue

Closes: #159
